org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerUtils.java

```diff
@@ -61,6 +61,30 @@ public static void sendClearedDirectoryLog(Path p) {
     Logger.getLogger(MAIN_LOGGER).info(eventData.toString());
   }
 
+  public static void logError(JavabuilderException error) {
+    LoggerUtils.logError(error.getExceptionMessage(), error.getLoggingString(), error.getCause());
+  }
+
+  public static void logError(
```

Comment:
Nice add! This is another one that would be nice to pull into a separate PR. (Not as necessary since it's a smaller, more straightforward change)